avogadro/qtplugins/templatetool/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 include(ExternalProject)
 
+find_package(Qt5 REQUIRED COMPONENTS Widgets OpenGL Svg)
```

Comment:
Widgets and OpenGL should already be covered for both Qt5 and Qt6

I think you want something like:
```
find_package(Qt${QT_VERSION} COMPONENTS Svg)
```